tests/Unit/Elliptic/DiscontinuousGalerkin/NumericalFluxes/FirstOrderInternalPenalty.py

```diff
@@ -0,0 +1,87 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+
+def average(int, ext):
+    return 0.5 * (int + ext)
+
+
+def jump(int, ext):
+    return int - ext
+
+
+def flux_for_field(grad_field, fluxes_argument):
+    return fluxes_argument * np.asarray(grad_field)
+
+
+def flux_for_auxiliary_field(field, fluxes_argument, dim):
+    return np.diag(np.repeat(fluxes_argument * field, dim))
+
+
+def normal_dot_numerical_flux_for_field(
+        n_dot_aux_flux_int, n_dot_aux_flux_ext,
+        div_aux_flux_int, div_aux_flux_ext,
+        fluxes_argument, penalty_parameter,
+        face_normal_int):
+    sigma = penalty_parameter
+    return np.dot(face_normal_int,
+                  average(
+                      flux_for_field(div_aux_flux_int, fluxes_argument),
+                      flux_for_field(div_aux_flux_ext, fluxes_argument)
+                  ) - sigma * jump(
+                      flux_for_field(n_dot_aux_flux_int, fluxes_argument),
+                      flux_for_field(-n_dot_aux_flux_ext, fluxes_argument)
+                  ))
+
+
+def normal_dot_numerical_flux_for_auxiliary_field(
+        n_dot_aux_flux_int, n_dot_aux_flux_ext,
+        div_aux_flux_int, div_aux_flux_ext,
+        fluxes_argument, penalty_parameter,
+        face_normal_int):
+    return average(n_dot_aux_flux_int, -n_dot_aux_flux_ext)
```

Comment:
Why the - sign here when Eq. (3) in the help text has a + sign? Maybe a comment to clarify?